README.md

```diff
@@ -1,60 +1,50 @@
-## Shop This City
+# Shop This City
+
   A mobile friendly application build on iOS Platform using React Native. It allows Users to favourite items from assorted well-known brands by their categories which enables the User to build up their own Pinterest Board. A divine feature that allows a User to go back to the items easily when the item has not been purchased. 
```

Comment:
Please redo this merge... it looks like you got rid of some important things? 

The screenshots are gone and some sections were deleted

If it was intentional ignore this comment.

Reply:
yup i made an animated gif to show the pictures :)